src/NServiceBus.Core/CircuitBreakers/RepeatedFailuresOverTimeCircuitBreaker.cs

```diff
@@ -28,7 +29,7 @@ public void Success()
             Logger.InfoFormat("The circuit breaker for {0} is now disarmed", name);
         }
 
-        public void Failure(Exception exception)
+        public Task Failure(Exception exception)
```

Comment:
[MSDN disagrees](https://msdn.microsoft.com/en-us/library/hh191443.aspx?f=255&MSPPError=-2147217396#BKMK_NamingConvention) with you :grinning: 

In the [Example](https://msdn.microsoft.com/en-us/library/hh191443.aspx?f=255&MSPPError=-2147217396#BKMK_CompleteExample) `AccessTheWebAsync()` is internal async method and still prefixed. I was under impression that access level doesn't matter, intent matters.